NoSQL/README.md

```diff
@@ -54,29 +54,79 @@ You can also use the Oracle NoSQL Command Line Interface (CLI). Start the
 following container (keep container `kvlite` running):
 
 ```shell
-$ docker run --rm -ti --link kvlite:store oracle/nosql:19.5-ce \
+
+$ docker run --rm -ti --link kvlite:store oracle/nosql:20.3  java -Xmx64m -Xms64m -jar lib/kvstore.jar version
```

Comment:
Why isn't the container configured to keep `kvlite` running by itself? Requiring manual intervention is an anti-pattern.

Reply:
I didn't understand

Reply:
Why are you telling folks to keep `kvlite` running? Why doesn't the container keep itself running? NoSQL is a daemon process, it should just run in the background continuously with no intervention required.

Reply:
Ok, this sentence was in the previous version - updated